src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/ParentTrait.php

```diff
@@ -16,6 +16,8 @@
 
 trait ParentTrait
 {
+    protected $allowParent;
```

Comment:
visibility change, should be reverted

Reply:
Same as above, the properties used by the trait were moved to the trait.

Reply:
two traits defining a same property cannot be used by the same class
so nope, it shouldn't

Reply:
Again, SHOULD traits share a property?

Having a property inside  a class is similar as being in concrete instead of abstract class, your abstraction depends on the implementation and vice-versa at the same time.

I mean, I can easily ignore these traits, but this seems broken. PHPStorm and PHPStan seem to agree.

Reply:
The code is perfectly fine as is, and intended as is. So the answer is YES :)